trajopt_sco/include/trajopt_sco/expr_ops.hpp

```diff
@@ -151,6 +151,11 @@ inline QuadExpr exprSub(QuadExpr a, const QuadExpr& b)
 }
 
 //////////////////////
+/**
+ * @brief Multiplies two AffExpr. Does not consider any optimizations for shared variables
+ * @return The QuadExpr result of the multiplication
+ */
+QuadExpr exprMult(const AffExpr&, const AffExpr&);
 
 QuadExpr exprSquare(const Var&);
 QuadExpr exprSquare(const AffExpr&);
```

Comment:
To simplify things should we remove the exprSquare since it is a spacial case of exprMult?

Reply:
We could, but I think the square probably gives you better performance if you know it is a square. It uses that knowledge to combine the like terms. E.g. (a+5)(a+5) = aa+10a+25 where multiply gives you (a+5)(a+5)=aa+5a+5a+25.

That being said, it may be simplified somewhere else in the optimizer and make no difference. But if so, I don't know where.